setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = krb5
-version = 0.7.0
+version = 0.7.0.post51
```

Comment:
You can leave this as it is, I've bumped the version as 0.6.0 is the latest in PyPI.

Reply:
leaving 0.7.0